Documentation/doc/Documentation/Installation.txt

```diff
@@ -334,6 +336,30 @@ cmake -DCGAL_HEADER_ONLY=ON . # configure \cgal
 
 and we do not need to run `make` anymore.
 
+\subsubsection subsection_headeronly_without_configuration Header-only without CMake Configuration
+
+Since \cgal 4.12, \cgal can be used in header-only mode, without even
+configuring \cgal\. Programs using \cgal (examples, tests, demos, etc.)
+must be directly configured using CMake. In this case, \cgal will be
+configured at the same time. The variable `CGAL_DIR` must point to the root
+directory of the \cgal source code (either the root of the unpacked release
+tarball, or the root of the Git working directory).
+
+So, using \cgal becomes now:
+
+<PRE>
+cd /path/to/your/code # go to the directory of the code source using \cgal
+cmake -DCGAL_DIR=<CGAL-root> .
+</PRE>
+
+\subsubsection subsection_headeronly_dependencies CGAL Dependencies
+
+\cgal can be used as a header-only library, but no all its dependencies are
```

Comment:
I do not know. Mael, would you agree with that change?

Reply:
I'm not completely familiar with header-only: if you're using GMP, then you can't use header-only? If so, then 'though' doesn't work because it implies that you can use header-only regardless.

Reply:
CGAL will be header-only: no compilation/configuration of CGAL, but your programs using CGAL will anyway require the GMP+MPFR libraries.

Reply:
So... maybe "though" is better: that is an issue, but it can be overcome.

Reply:
then 'though' is good, yes.